### PR TITLE
v0.1.8

### PR DESCRIPTION
**Pull request by MandarinDuck**
Expanding to use highmem node.

### CODE REVIEW

File: viper.py [hunk 1]
```diff
@@ -32,19 +30,55 @@
...
-        labels= {"viper": dag.dag_id},
+        labels= {"viper": viper_dag.dag_id},
         name= "viper",
         in_cluster= True,
         task_id= "viper",
         get_logs= True,
         startup_timeout_seconds= 500,
-        annotations= {"iam.amazonaws.com/role": ROLE},
+        annotations= {"iam.amazonaws.com/role": VIPER_ROLE},
+        tolerations=[
```
Comment (samtazzyman) [inline viper.py]: excuse the ignorance: what's this doing, please?

File: viper.py [hunk 2]
```diff
@@ -32,19 +30,55 @@
 )
 
 viper_task = KubernetesPodOperator(
-        dag= dag,
+        dag= viper_dag,
```
Comment (samtazzyman) [inline viper.py]: why the change from `dag` to `viper_dag`? (ditto for `IMAGE` and `VIPER_IMAGE`)

File: viper.py (hunk 1 above)
Comment (MandarinDuck) [reply viper.py]: I believe it's instructing Kubernetes to use the highmem node specifically for this airflow job (so that viper can run).

Comment (MandarinDuck) [reply viper.py]: Code came via Ravi

File: viper.py (hunk 2 above)
Comment (MandarinDuck) [reply viper.py]: I've renamed the variables because I've got a plan in mind to chain together a DAG from separate DAGs such that you can run a full nomis-extract >> nomis-transform >> viper >> other stuff DAG by calling their separate DAGs (thereby enabling them to still be run separately without code redundancy). Not sure if the rename is actually necessary yet re. execution scope,  but it will aid in clarity if I do any copy and paste.

Comment (MandarinDuck) [reply viper.py]: I don't know if it will break the concourse checks but I've seen them named differently in other DAGs like the PQ one.

Comment (samtazzyman) [reply viper.py]: makes sense

File: viper.py (hunk 1 above)
Comment (samtazzyman) [reply viper.py]: yup, makes sense
